README.md

```diff
@@ -5,7 +5,9 @@ Bitcoin Core integration/staging tree
 
 https://bitcoincore.org
 
-What is Bitcoin?
+![Ronnie Swoleman](https://github.com/JNKHunter/bitcoin/blob/swole/doc/coleman_large.jpg?raw=true)
```

Comment:
I think this is going to blow up, we should use a static cdn, @JNKHunter what do you think?